docs/modules/ROOT/pages/advanced-networking.adoc

```diff
@@ -0,0 +1,26 @@
+= Advanced Network Configuration
+:description: In Hazelcast, you can configure the Hazelcast members with separate server sockets using a different network configuration for different protocols. It is called Advanced Network and support Member, Client, Rest, Memcache and WAN protocols.
+
+In the Hazelcast Platform Operator, advanced networking configuration is enabled by default and Rest and Client protocols are already configured. For Client protocol the port number is 5701. For Rest, port number is 8081 and `Persistence`, `HealthCheck` and `ClusterWrite` endpoint groups enabled by default.
```

Comment:
Using list might be good idea to emphasizes current config

.... configured:
-  For Client protocol, the port number is **5701**. 
-  For REST, the port number is **8081**, and `Persistence`, `HealthCheck`, and `ClusterWrite` endpoint groups are enabled by default.